pkg/apis/ark/v1/backup.go

```diff
@@ -203,6 +203,10 @@ type BackupStatus struct {
 	// VolumeSnapshotsCompleted is the total number of successfully
 	// completed volume snapshots for this backup.
 	VolumeSnapshotsCompleted int `json:"volumeSnapshotsCompleted"`
+
+	// VolumeSnapshotsFailed is the total number of fialed volume
+	// snapshots for this backup.
+	VolumeSnapshotsFailed int `json:"volumeSnapshotsFailed"`
```

Comment:
I don't think we necessarily need this recorded here - I assumed we would just calculate it for the metrics by doing the subtraction. @skriss wdyt?

Reply:
Yeah I think we can just calculate it, no need for API field

Reply:
Alright, fixing it :)